### PR TITLE
Lvpyt 7 player crud fixed

### PR DESCRIPTION
**Pull request by romajarik**

### CODE REVIEW

File: itaplay/itaplay/player/apps.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```
Comment (ypyrko) [inline itaplay/itaplay/player/apps.py]: Why do you need this file?

File: itaplay/itaplay/player/forms.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django import forms
+from models import Player
+
+class PlayerForm(forms.Form):
+        class Meta:
+            model = Player
+            fields = ('id','player_name', 'player_description', 'player_mac_address', "player_status",)
```
Comment (ypyrko) [inline itaplay/itaplay/player/forms.py:7]: What type of this fields?

File: itaplay/itaplay/player/views.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+from django.views.generic.base import View
+from django.views.generic import DeleteView
+from django.views.decorators.csrf import csrf_protect
+from django.shortcuts import render, render_to_response
+from django.http import HttpResponseRedirect, HttpResponse
+
+
+class PlayerView(View):
+    def get(self, request, player_id=None):
+        if player_id==None:
```
Comment (ypyrko) [inline itaplay/itaplay/player/views.py]: if player_id==None: -> if not player_id:

File: itaplay/itaplay/static/js/app/player/views/all_player.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        <div>
+            <md-button flex="500"  class="md-primary md-raised" href="#player/add_new" >Add new player</md-button> 
+        </div>	
+        <div flex=20></div>
+    </div>
+    <div layout="row">
+        <div flex=15></div>
+        <div flex=70  layout-padding>
+            <div  layout-padding ng-cloak ng-repeat = "player in players | filter:name">
+                <md-card style="background-color: #CCD9D7;"  md-theme-watch layout="row">
```
Comment (ypyrko) [inline itaplay/itaplay/static/js/app/player/views/all_player.html:20]: move styles too *.css

File: itaplay/itaplay/player/apps.py (hunk 1 above)
Comment (romajarik) [reply itaplay/itaplay/player/apps.py]: Sorry, my fault. We decided to use INSTALLED_APPS

**Comment by vova-kul** [comment]
test jenkins bot

**Comment by vova-kul** [comment]
Can one of the admins verify this patch?

**Comment by vova-kul** [comment]
ok to test

**Comment by vova-kul** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**